src/main/kotlin/com/bridgecrew/utils/Constants.kt

```diff
@@ -47,3 +47,5 @@ val SUPPRESSION_BUTTON_ALLOWED_FILE_TYPES: Set<FileType> = setOf(
         FileType.YAML,
         FileType.TERRAFORM
 )
+
+val CUSTOM_POLICIES_TO_BE_IGNORED = listOf<String>("yaml policy  secrets", "alapaka", "Copy of S3 bucket MFA Delete is not enabled")
```

Comment:
Need to check if there's any security risk with having explicit policies in an open source code base (I don't know if there is)

Reply:
I think it's just the name so I don'y know if we expose too much here...